### ISSUE TITLE
Points tracker

### ISSUE THREAD

**Issue by ExerciseOSRS** [LlemonDuck/tombs-of-amascut#38]
Initially it was working fine but I just had a deathless 400 invo raid which usually tells me around 10-13% (if using the wiki calc it shows similar) but this raid was showing 2.2% at the end? 23,000 points no deaths when on screenshots i have from earlier raids when the feature was released it was showing me as 4.5% at only 11k points so I'm not sure what's happened there. 
provided a screenshot, not sure how/if it'll help. 
https://gyazo.com/88565022799ca972baa88e21d48b7f0e

**Comment by LlemonDuck** [comment]
This looks like it calculated your drop chance on a 0 raid level. I released a new version of plug in a few hours ago that addresses this issue. Can you check whether you are on the most up to date version? 

If your client is still open, you can check by opening the plugin hub panel and seeing if there is an update button for Tombs of Amascut.

**Comment by ExerciseOSRS** [comment]
Yeah my client was open for a while, I restarted it and saw the plugin address the updates and it seems to be working again now, thanks for the amazing work.
